students/jeff_shabani/session03/mailroom.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+import sys
+
+from operator import itemgetter
+from collections import OrderedDict
+
+DONORS = {'William B': [120, 130, 50],
+          'Sammy Maudlin': [500, 125, 670, 1000],
+          'Bobby Bittman': [10],
+          'Skip Bittman': {75, 125, 19},
+          'Ashley Lashbrooke': [10000, 15000]}
+
+prompt = "\n".join(("Welcome to my charity!",
+          "Please select and option below:",
+          "1 - Send a Thank You",
+          "2 - Create a Report",
+          "3 - Quit",
+          ">>> "))
+
+def view_donor_names():
+    for k,v in DONORS.items():
+        print(k)
+
+def add_new_donor(name, donor_list):
+    amount = int(input(f'How much would this donor like to donate?'))
+    donor_list[name] = [amount]
+
+
+
+def get_a_name():
```

Comment:
I would rename this to something that describes what it does.  `get_a_name` isn't it :)